inlong-sort/pom.xml

```diff
@@ -48,7 +48,7 @@
         <rat.basedir>${basedir}</rat.basedir>
         <hbase.version>2.2.3</hbase.version>
         <dlc.hive.version>2.3.7</dlc.hive.version>
-        <iceberg.hive.version>2.3.7</iceberg.hive.version>
+        <iceberg.hive.version>3.1.3</iceberg.hive.version>
```

Comment:
Hive 2.3.7 has security risks. Could it be upgraded in Sort Connector?

@EMsnap @thesumery check it, please.